packages/jmix-react-antd/src/ui/form/controls/RadioButton.tsx

```diff
@@ -0,0 +1,31 @@
+import { Radio } from "antd";
+import * as React from "react";
+import { JmixFormFieldWrapper } from "./base/JmixFormFieldWrapper";
+import { JmixFormFieldProps } from "./base/JmixFormFieldProps";
+import { RadioGroupProps } from "antd/es";
+
+export function RadioButton({
+  entityName,
+  propertyName,
+  formItemProps,
+  value,
+  onChange,
+  ...rest
+}: JmixFormFieldProps & RadioGroupProps) {
+  return (
+    <JmixFormFieldWrapper
+      entityName={entityName}
+      propertyName={propertyName}
+      formItemProps={formItemProps}
+      renderField={isReadOnly => (
+        <Radio.Group
+          value={value}
+          onChange={onChange}
+          disabled={isReadOnly}
+          optionType="button"
```

Comment:
Let's make it radio button by default.

Reply:
![image](https://user-images.githubusercontent.com/8298981/144018581-842e6156-7c7d-4d7b-8d21-6a39643a9fd8.png)